### PR TITLE
Dev nick

### PR DESCRIPTION
**Pull request by NickIs211**

### CODE REVIEW

File: app/controllers/notification_controller.rb [hunk 1]
```diff
@@ -7,7 +7,7 @@ def show
     notification.save
 
     if notification.notifiable_type == 'TutoringSessionUser'
-      tutoring_session_user_notification
+      tutoring_session_user_notification # redirect when given a TutoringSessionUser Noti
```
Comment (baileyjrob) [inline app/controllers/notification_controller.rb:10]: Good job adding documentation

File: app/views/users/show_schedule.html.erb [hunk 2]
```diff
@@ -1,23 +1,42 @@
...
+          <% end %>
+        </tbody>
+      </table>
+    </div>
+  </div>
+  <script>
+  $(function() {
+    $('#schedule').DataTable();
+  });
+  </script>
```
Comment (baileyjrob) [inline app/views/users/show_schedule.html.erb:41]: Why do we need to add JS? It was working fine without it before.

Comment (NickIs211) [reply app/views/users/show_schedule.html.erb:41]: The JS is there to make the table more pretty + add the ability to search for a specific session

**Review by baileyjrob — APPROVED** [review]
Everything looks good

**Pull request merged**